docker-compose.yml

```diff
@@ -1,15 +1,18 @@
 services:
   postgres:
     image: postgres:16
+    container_name: hostedgpt_postgres
     restart: always
     environment:
-      POSTGRES_USER: app
-      POSTGRES_DB: app_development
-      POSTGRES_PASSWORD: secret
+      # NOTE: If you use DATABASE_URL, it needs to match these env vars
+      #  so you can either have it match the default values for these env vars
+      #  or you can set these env vars to match the user and password from the url
+      - POSTGRES_USER=${HOSTEDGPT_DB_USERNAME:-hostedgpt_user}
+      - POSTGRES_PASSWORD=${HOSTEDGPT_DB_PASSWORD:-secret}
     volumes:
       - hostedgpt_pgdata:/var/lib/postgresql/data
     healthcheck:
-      test: ["CMD", "pg_isready", "-U", "app", "-d", "app_development"]
```

Comment:
doesn't seem necessary to have the healthcheck try to connect to the database specifically. Just the database server seems like enough. This way we don't have to put the database name here and have another place to update it